ctrlp.py

```diff
@@ -1,4 +1,9 @@
 # coding: utf-8
+# @author msm
+# @category Search
+# @keybinding ctrl P
```

Comment:
I'm a bit concerned about overriding the default keybinging with a script. Even though IMO default Ctrl+P is useless (and if you want it, you can always type `act print` in command palette).

OTOH maybe this plugin could automatically unbind the default print shortcut. This would simplify installation somewheat, I guess. I'll merge this PR without this override, and we can decide on something later.

```suggestion
```